packages/components/src/globals/scss/_css--reset.scss

```diff
@@ -22,9 +22,9 @@
     font-family: inherit;
     vertical-align: baseline;
 
-    & > *,
-    & > *:before,
-    & > *:after {
+    & *,
+    & *:before,
+    & *:after {
```

Comment:
since this is nested we don't need the ampersands right?

```css
    *,
    *::before,
    *::after {
```

Reply:
True! I'll remove those as well